packages/generators/templates/index.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Application entry point
```

Comment:
IMO this comment is not required.
Instead, we can have a README.md at user's root to guide users about each file. This is also followed by other scaffolding tools.

Reply:
You are right!